zsh-abbr.zsh

```diff
@@ -36,7 +36,7 @@ typeset -gi ABBR_QUIET=${ABBR_QUIET:-0}
 typeset -gi ABBR_QUIETER=${ABBR_QUIETER:-0}
 
 # Temp files are stored in
-typeset -g ABBR_TMPDIR=${ABBR_TMPDIR:-${${TMPDIR:-/tmp}%/}/zsh-abbr/}
+typeset -g ABBR_TMPDIR=${ABBR_TMPDIR:-${${TMPDIR:-/tmp}%/}/zsh-abbr-${UID:-${USER}}/}
```

Comment:
If we make this change, I think it will be cleaner to make sure there's no trailing slash in this line. Then subsequent uses can be `${ABBR_TMPDIR}/` instead of `${ABBR_TMPDIR%/}/`.